sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -70,6 +70,25 @@ def get_namespaced_custom_object_response(*args, **kwargs):
     return mock_thread
 
 
+def list_namespaced_custom_object_response(*args, **kwargs):
+    if args[2] == "timeout":
+        raise multiprocessing.TimeoutError()
+    elif args[2] == "runtime":
```

Comment:
Please can you use constants here: https://github.com/kubeflow/training-operator/blob/522e8d9f89112fa2718fa7e257865576c77c6181/sdk/python/kubeflow/training/api/training_client_test.py#L28-L29.

Additionally, we can change them here: https://github.com/kubeflow/training-operator/blob/522e8d9f89112fa2718fa7e257865576c77c6181/sdk/python/kubeflow/training/api/training_client_test.py#L58

Reply:
Sure! Added.